docs/resources/Channel.md

```diff
@@ -562,6 +597,23 @@ once per 100 milliseconds, but will downsample so that no more than 256 datapoin
 As of 2023-04-14, clients upload a 1 channel, 48000 Hz, 32kbps Opus stream in an OGG container.
 The encoding, and the waveform details, are an implementation detail and may change without warning or documentation.
 
+
+### Message Snapshot Object
+
+#### Message Snapshot Structure
+
+| Field      | Type      | Description                                                                      |
+|------------|-----------|----------------------------------------------------------------------------------|
+| message \* | [message] | subset of fields in the [message object](#DOCS_RESOURCES_CHANNEL/message-object) |
+| guild_id?  | snowflake | ID of the origin message's guild                                                 |
```

Comment:
TODO: this has changed to ```json
`"guild": { "id": 1234 }`

Reply:
we've decided to actually not return this and instead direct people to use the `message.message_reference` of the parent